### PR TITLE
Add notes about disputes

### PR DESCRIPTION
**Pull request by maltekliemann**
Just getting the ball rolling on getting the docs up to speed.

### CODE REVIEW

**Comment by maltekliemann** [comment]
@Chralt98 Can you take a quick peek at this and tell me if this is still required or if there's any overlap?

File: docs/learn/using-zeitgeist-markets.md [hunk 1]
```diff
@@ -239,43 +239,35 @@ within a fixed frame of time. If the oracle fails to submit the report in time,
...
 ### Disputes
 
-Every time a report is submitted (by the oracle or a stand-in), the market is
-not resolved for another 24 hours. During this period of time, other users can
-submit a _dispute_ if they believe that the report is incorrect.
+When a report is submitted (by the oracle or a stand-in), the market is not
+resolved for a certain window of time called the _dispute period_. During this
+period of time, other users can submit a _dispute_ if they believe that the
+report is incorrect. To do so, they stake a certain amount of ZTG, the _dispute
+bond_ and report the outcome they believe to be correct (the _suggestion_).
```
Comment (Chralt98) [inline docs/learn/using-zeitgeist-markets.md:251]:
```suggestion
bond_.
```

File: docs/learn/using-zeitgeist-markets.md [hunk 2]
```diff
@@ -239,43 +239,35 @@ within a fixed frame of time. If the oracle fails to submit the report in time,
...
 
-To do so, they stake some ZTG and report the outcome they believe to be correct.
-The 24 hour window for disputes is then reset, and other users (including the
-oracle) can dispute the new report.
+How the dispute is resolved depends on what _dispute mechanism_ was selected
+during market creation. The currently available dispute mechanisms are:
```
Comment (Chralt98) [inline docs/learn/using-zeitgeist-markets.md:254]:
```suggestion
How the dispute is resolved depends on the [decentralized court] _dispute mechanism_.
```

File: docs/learn/using-zeitgeist-markets.md [hunk 3]
```diff
@@ -239,43 +239,35 @@ within a fixed frame of time. If the oracle fails to submit the report in time,
...
 
-If the dispute cannot be resolved, it is escalated to the
-[Decentralized Court](./court.md).
+- _Authorized_: After the market has been disputed, the [Advisory Committee]
+  decides which outcome the market will resolve to.
+- _[Decentralized Court]_ (currently disabled): A group of randomly selected
+  jurors vote secretly to find a [Schelling point] among the market's outcomes,
+  which is then used as the resolved outcome.
```
Comment (Chralt98) [inline docs/learn/using-zeitgeist-markets.md:260]:
```suggestion
In short: A group of randomly selected jurors vote secretly to find a [Schelling point] among the market's outcomes, which is then used as the resolved outcome.
```

File: docs/learn/using-zeitgeist-markets.md [hunk 4]
```diff
@@ -239,43 +239,35 @@ within a fixed frame of time. If the oracle fails to submit the report in time,
...
-:::important
-During the Beta campaign, only _simple disputes_ are enabled. This means that
-the outcome of a market can be disputed a maximum of six times (with a 24h
-window, except the last one), each time with a higher stake. The report of the
-sixth dispute will be used to resolve the market. There is no decentralized
-court in the beta.
-:::
+If the dispute is found to have been justified in the sense that the market
+resolves to the outcome suggested by the disputant, the dispute bond is returned
+to the disputant. If this is not the case, the dispute bond is slashed.
```
Comment (Chralt98) [inline docs/learn/using-zeitgeist-markets.md:264]:
```suggestion
to the disputant. If this is not the case, the dispute bond is slashed.
More on that in the bonds section of the [Using Zeitgeist Markets] page.
```

File: docs/learn/using-zeitgeist-markets.md [hunk 5]
```diff
@@ -343,6 +335,9 @@ outcome assets and ZTG) or other individual assets to any existing pool.
 
 <!-- Links -->
 
+[advisory committee]: ./governance.md#advisory-committee
+[decentralized court]: ./court
+[schelling point]: https://en.wikipedia.org/wiki/Focal_point_(game_theory)
```
Comment (Chralt98) [inline docs/learn/using-zeitgeist-markets.md:340]:
```suggestion
[schelling point]: https://en.wikipedia.org/wiki/Focal_point_(game_theory)
[Using Zeitgeist Markets]: ./using-zeitgeist-markets
```

**Review by Chralt98 — CHANGES_REQUESTED** [review]

**Comment by Chralt98** [comment]
I didn't check the non-modified lines of this file. I think if we just mention court as the dispute mechanism we are fine for the future, because why mentioning (soon) deprecated code?

**Review by Chralt98 — APPROVED** [review]

**Pull request merged**